src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -234,7 +241,21 @@ const ContentMessageComponent: React.FC<ContentMessageProps> = ({
           />
         ))}
 
-        {failedToSend && <FailedToSendWarning failedToSend={failedToSend} knownUsers={conversation.allUserEntities} />}
+        {failedToSend && (
+          <PartialFailureToSendWarning failedToSend={failedToSend} knownUsers={conversation.allUserEntities} />
+        )}
+
+        {status === StatusType.FAILED && (
+          <CompleteFailureToSendWarning
+            isTextAsset={message.getFirstAsset().isText()}
+            onDiscard={() => {
+              onDiscard();
+            }}
+            onRetry={() => {
+              onRetry(message);
+            }}
```

Comment:
```suggestion
            onRetry={() => onRetry(message)}
```
WDYT?

Reply:
forgot to remove dem curly braces, good catch